libraries/app/api.cpp

```diff
@@ -328,6 +328,10 @@ namespace graphene { namespace app {
              start = node.operation_id;
        } catch(...) { return result; }
 
+       auto es = _app.get_plugin<elasticsearch::elasticsearch_plugin>("elasticsearch");
```

Comment:
This will throw if the plugin is not enabled, that's probably not what you want. I suppose that's also the reason why the travis tests failed.

Reply:
What about having the `get_plugin` option in a `try` block and do the normal non elasticsearch code in the `catch`?

I will of course prefer something else but i cant find a way to check this without this problem.

Reply:
Doing try/catch for this feels dirty. Maybe add an `is_plugin_enabled` call to `application`?